fastlane/Appfile

```diff
@@ -0,0 +1,3 @@
+app_identifier(Constants.BUNDLE_ID_PRODUCTION) # The bundle identifier of your app
```

Comment:
I think we don't need this `app_identifier` as a global variable. We always use `app_identifier` in DistributeManger, so I think it's safe to remove this variable here

Reply:
This will be use when we call `match` for both CI and local but it actually does not make sense to put the value here. I fixed missing values and remove this file.